html_app/ui/microscopy.soy

```diff
@@ -864,46 +782,47 @@
+            cell_treatment_id='{$cell_treatment.id}' microscopy_id='{$microscopy.id}'
+            assignment_id="{$assignment.id}" experiment_id="{$experiment.id}" value="{$k}"
+            lane_kind="{$row.kind}" lane_id="{if $row.kind == 'existing'}{$row.lane.id}{/if}">{$kinds[$k].name}</span>
+        {/foreach}
 {else}
 	<select class="scb_f_microscopy_select_slide_type" cell_treatment_id='{$cell_treatment.id}' role='select'
 			microscopy_id='{$microscopy.id}' assignment_id="{$assignment.id}" experiment_id="{$experiment.id}"
-			lane_kind="{$lane.kind}" lane_id="{if $lane.kind == 'existing'}{$lane.lane.id}{/if}"
-	{if $lane.is_sample_enabled}{else}disabled="disabled"{/if}
+			lane_kind="{$row.kind}" lane_id="{if $row.kind == 'existing'}{$row.lane.id}{/if}"
+	        {if $row.is_sample_enabled}{else}disabled="disabled"{/if}
 	>
-	{if $lane.kind == 'existing'}
-		{if $lane.cell_treatment.cell_line == $assignment.template.model.microscopy.valid[0] and $lane.cell_treatment.treatment_list.first.drug_list.list[0].drug_id == $assignment.template.model.microscopy.valid[1]}
-			{foreach $k in keys($assignment.template.slide_parser[$lane.cell_treatment.treatment_list.first.collection_id])}	
+
+	{if $row.kind == 'existing'}
+		{if $row.cell_treatment.cell_line == $assignment.template.model.microscopy.valid[0] and $row.cell_treatment.treatment_list.first.drug_list.list[0].drug_id == $assignment.template.model.microscopy.valid[1]}
+			{foreach $k in keys($assignment.template.slide_parser[$row.cell_treatment.treatment_list.first.collection_id])}
 			<option value='{$k}' role='option'
-			{if $lane.lane.kind == $k}selected="selected"{/if}>{$kinds[$k].name}</option>
+			{if $row.lane.kind == $k}selected="selected"{/if}>{$kinds[$k].name}</option>
 			{/foreach}
 		{else}
 			{foreach $k in keys($assignment.template.slide_parser.default)}
 			<option value='{$k}' role='option'
-			{if $lane.lane.kind == $k}selected="selected"{/if}>{$kinds[$k].name}</option>
+			{if $row.lane.kind == $k}selected="selected"{/if}>{$kinds[$k].name}</option>
 			{/foreach}
 		{/if}
-		
 	{else}
+	<!--row not existing, want to show Select Analysis option-->
 	{if length(keys($kinds)) != 1 or length(keys($kinds.IF.conditions)) != 1 }
 	<option  role='option' selected="selected" disabled="disabled" value=''>Select Analysis</option>
 	{/if}
-	{if $lane.cell_treatment.cell_line == $assignment.template.model.microscopy.valid[0] and $lane.cell_treatment.treatment_list.first.drug_list.list[0].drug_id == $assignment.template.model.microscopy.valid[1]}
-		{foreach $k in keys($assignment.template.slide_parser[$lane.cell_treatment.treatment_list.first.collection_id])}	
+	<!-- Comparing to VALID-->
+	{if $row.cell_treatment.cell_line == $assignment.template.model.microscopy.valid[0] and $row.cell_treatment.treatment_list.first.drug_list.list[0].drug_id == $assignment.template.model.microscopy.valid[1]}
```

Comment:
I don't know soy enough to know, but would be nice to have this on multiple lines